### PR TITLE
adding comments and documentation

### PR DESCRIPTION
**Pull request by olhabn**

### CODE REVIEW

File: src/foam/nanos/column/CommonColumnHandler.js [hunk 1]
```diff
@@ -91,6 +91,14 @@ foam.CLASS({
       return  prop ? prop.property[property] : of.getAxiomByName(this.returnPropertyNamesForColumn(colObj))[property];
     },
     function groupObjectsThatAreRelatedToNestedProperties(of, arrayOfNestedPropertiesName, arrayOfValues) {
+      //this function creates obj for nested properties and adds it to the map
```
Comment (kgrgreer) [inline src/foam/nanos/column/CommonColumnHandler.js]: I know it's a minor point, but could you please leave a space after //'s. Thx

File: src/foam/nanos/google/api/sheets/Google_Sheets_Data_Services.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#Google export related services 
+GoogleApiAuthService takes care of setting up authorization flow with a GoogleApiCredentials.
+GoogleDriveService takes care of making calls to Google Drive APIs, with which we can create files and directories.
+GoogleSheetsApiService takes in data that we want to export and its metadata (which type of the cell should be used to display values) and sends requests to GoogleSheetsApi to create new sheet or copy existing one (which we do there is a template that user would like to use).
+
+#Google Sheets template
+To create a Google Sheet templete:
```
Comment (kgrgreer) [inline src/foam/nanos/google/api/sheets/Google_Sheets_Data_Services.md]: templete -> template

File: src/foam/nanos/google/api/sheets/Google_Sheets_Data_Services.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+1. Create a new sheet with Google Sheets or use excisting one
+2. Leave first sheet empty. There should be nothing in there
+3. Add your formula, charts and any other components of template on sheets other then first one
+4. Make your sheet accesible to anyone
+5. Go to back office -> Admin -> Report Template -> Create new template
+6. For id use your google sheet id
+for dao use dao that is used by a table which you want to export to Google Sheets
```
Comment (kgrgreer) [inline src/foam/nanos/google/api/sheets/Google_Sheets_Data_Services.md]: dao -> DAO

File: src/foam/nanos/google/api/sheets/Google_Sheets_Data_Services.md (hunk 2 above)
Comment (olhabn) [reply src/foam/nanos/google/api/sheets/Google_Sheets_Data_Services.md]: fixed

File: src/foam/nanos/google/api/sheets/Google_Sheets_Data_Services.md (hunk 3 above)
Comment (olhabn) [reply src/foam/nanos/google/api/sheets/Google_Sheets_Data_Services.md]: fixed

File: src/foam/nanos/column/CommonColumnHandler.js (hunk 1 above)
Comment (olhabn) [reply src/foam/nanos/column/CommonColumnHandler.js]: sure! fixed. thanks

**Pull request merged**